guide/start/policies.md

```diff
@@ -355,8 +355,8 @@ Tomcat on the vagrant VMs named "byon1" to "byon4":
 
 {% highlight bash %}
 $ for i in byon{1..4}; do
-$   vagrant ssh ${i} --command 'ps aux | grep -i tomcat |  grep -v grep | awk '\''{print $2}'\'' | xargs kill -9'
```

Comment:
clipboard support is working fine for me -- just click the icon

would be nice to exclude it from a manual copy/paste but that's harder.  gonna experiment with http://stackoverflow.com/questions/22159929/html-exclude-text-from-clipboard-copy